test/functional/test_framework/test_node.py

```diff
@@ -488,6 +489,7 @@ def add_p2p_connection(self, p2p_conn, *, wait_for_verack=True, **kwargs):
             kwargs['dstaddr'] = '127.0.0.1'
 
         p2p_conn.peer_connect(**kwargs, net=self.chain)()
```

Comment:
```suggestion
        p2p_conn.peer_connect(**kwargs, net=self.chain, factor=self.factor)()
```

Reply:
In some previous variant of my code I encountered such a problem. Maybe just because I was still writing it. Right now, it seems that all those conversions aren't needed anymore. I'm just executing some tests and they're all passing.

Reply:
I don't understand this change?

Reply:
Ah, yes, now I see it. It's being forwarded to derived class P2PInterface!